specification/ai/Azure.AI.Projects/red-teams/models.tsp

```diff
@@ -122,6 +122,26 @@ union RiskCategory {
   UngroundedAttributes: "UngroundedAttributes",
```

Comment:
`UngroundedAttributes`, `CodeVulnerability` and `ProtectedMaterial` are not yet supported by SDK

Reply:
+1